### ISSUE TITLE
Pop-over displays too low on on first attempt if there is not enough vertical space below the post

### ISSUE THREAD

**Issue by wiggin77** [mattermost/mattermost-plugin-github#815]
There is an issue where the GitHub pop-over shows too low and is unreadable if the post is too low in the thread and there is not enough vertical space to show it.

It seems on the 1st attempt, the pop-over will always display below the post. On the 2nd attempt it will correctly display upward as it does not fit below. This should always happen.

First pop-over attempt:
![jira_1](https://github.com/user-attachments/assets/8782be3c-5c13-4352-8d16-3a0e4350fbd6)

 
Second pop-over attempt:
![jira_2](https://github.com/user-attachments/assets/b7b8f746-c8b6-4b0b-9883-0b0ace354d1f)

 

**Note**: I (@DHaussermann ) am unsure if this can be fixed in GitHub or if it's a webapp issue.

Steps:

- Open a channel in Mattermost

- Make a post with a GitHub link ex: https://github.com/mattermost/mattermost-plugin-apps/pull/230

- Hold your mouse pointer over the link
**Observed** The pop-over is displayed downward and cannot be read.  

See https://mattermost.atlassian.net/browse/MM-37643

**Comment by raghavaggarwal2308** [comment]
@wiggin77 I don't think this can be fixed in the Github plugin as we are just registering a tooltip component in the plugin and the rest is handled by the MM webapp.
